src/pudl/etl/__init__.py

```diff
@@ -12,7 +12,6 @@
     AssetsDefinition,
     AssetSelection,
     Definitions,
-    ExperimentalWarning,
```

Comment:
Random import that was no longer being used. Not sure why Ruff didn't zap it before.